packages/rspack-cli/tests/build/basic/basic.test.ts

```diff
@@ -60,4 +60,21 @@ describe("build command", () => {
 		expect(mainJs).toContain("other");
 		expect(mainJs).not.toContain("CONFIG");
 	});
+	it("output-path option should have higher priority than config", async () => {
+		const { exitCode, stderr, stdout } = await run(__dirname, [
+			"--output-path",
+			"public",
```

Comment:
This field results in the creation of a `﻿public` directory that isn't ignored by git. This might cause confusion and potentially degrade the developer experience.